### PR TITLE
Blog app - creating a data model

### PR DESCRIPTION
**Pull request by brainskev**
In  this branch I achieved the following;

- Created a new app. Named it Blog-App
- Initialized my project with Git.
- Made sure that my project has Postgres database set up.
- Build my project schema.
- Created and run the necessary migration files.
- Made sure Table and column names match the ERD diagram.
- Ensured Foreign keys are included.
- Ensured All columns that are foreign keys have a corresponding index.

### CODE REVIEW

File: db/migrate/20230608075933_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.integer :bio
```
Comment (cynthiainga) [inline db/migrate/20230608075933_create_users.rb]: - [ ] Good job so far but the bio is not an integer, integer refers to numbers. `Bio` should be a text, please kindly fix that 🙏🏾

File: db/migrate/20230608081704_create_posts.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :author, null: false, foreign_key: { to_table: :users }
+      t.string :title
+      t.text :text
+      t.integer :comment_counter, default: 0
+      t.integer :likes_counter, default: 0
```
Comment (cynthiainga) [inline db/migrate/20230608081704_create_posts.rb]: - [ ] Based on the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) each table has a `UpdateAt` and `CreatedAt` columns and all your tables doesn’t have those columns. All the tables should have `t.timestamps` that create `updated_at and created_at columns`. Kindly fix that 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @brainskev, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Database adapter is using Postgresql ✔️ 
- [ ] Foreign keys are included ✔️ 
- [ ] All columns that are foreign keys have a corresponding index ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can get the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by brainskev** [comment]
Hi @cynthiainga, @ichala 

Just wanted to drop a quick note to say thank you for reviewing my code. Your feedback and suggestions have been incredibly valuable, and I truly appreciate your time and expertise. I would like to connect with you on [GitHub ](https://github.com/brainskev) & [LinkedIn](https://www.linkedin.com/in/kelvinkaviku/) to stay in touch and potentially collaborate on future projects. Your contributions have been invaluable, and I believe we can learn a lot from each other.

Looking forward to connecting!

Best regards,
Kelvin Kaviku 
@brainskev

**Pull request merged**